source/2019-08-15-octane-release-plan.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Octane is coming in v3.14
+author: Yehuda Katz, Jen Weber
+tags: 2019, Announcement, Recent Posts
+responsive: true
+---
+
+The Ember community is wrapping up the [Octane](https://emberjs.com/editions/) edition, and we expect to get it over the finish line over the next three releases!
+
+- The latest release of Ember is 3.12, which is a an LTS (long term support) candidate. As an LTS, it does not have any significant new features, but it does have many of Octane's features! Check out the [release blog posts](https://blog.emberjs.com) to learn about them.
+- Ember 3.13 will be feature-complete for Octane, and apps using stable Ember can opt in to Octane. At this point, we're still wrapping up polish, especially in the codemods, inspector and guides.
+- In Ember 3.14, Octane mode will be the primary, recommended way to use Ember. Ember 3.14 will default new apps to Octane mode, and it will be the recommended mode. The guides and tutorials will document Octane mode and Octane idioms, and codemods to help users migrate to Octane idioms will be available.
```

Comment:
"Octane mode" means specifically enabling the right optional features?

Reply:
This part was a little confusing for me because so far in the post, "Octane mode" has not been defined (as a separate concept from just "Octane"). Will the concept of a "mode" be meaningful to an Ember developer reading this post? Where would they have heard this term before?

Reply:
Edited to remove "mode"

- The next release of Ember is 3.12, which is a an LTS (long term support) candidate. Check out the [release blog posts](https://blog.emberjs.com) to learn which of Octane's features are already in stable releases.
- Ember 3.13 will be feature-complete for Octane, and apps using stable Ember can opt in. At this point, we're still wrapping up polish, especially in the codemods, inspector and guides.
- In Ember 3.14, Octane will be the primary, recommended way to use Ember. In Ember 3.14, new apps will have Octane's optional features enabled by default. The guides and tutorials will show Octane examples, and codemods will be available to help users migrate to Octane idioms.